SentinelOne/sentinelone_module/logs/configuration.py

```diff
@@ -0,0 +1,16 @@
+from typing import Optional
+from uuid import UUID
+
+from sekoia_automation.connector import DefaultConnectorConfiguration
+
+
+class SentinelOneLogsConnectorConfiguration(DefaultConnectorConfiguration):
+    client_secret: str
+    client_id: str
+    api_key: str
+    management_domain: str
+    uuid: UUID
+    intake_key: str
```

Comment:
Already defined in [DefaultConnectorConfiguration](https://github.com/SEKOIA-IO/sekoia-automation-sdk/blob/main/sekoia_automation/connector/__init__.py#L40)
```suggestion
```